validator/validator-main.protoascii

```diff
@@ -2735,6 +2735,18 @@ tags: {
   mandatory_ancestor: "FORM [method=POST]"
   spec_url: "https://amp.dev/documentation/components/amp-form/"
 }
+tags: {
+  html_format: AMP
+  tag_name: "INPUT"
```

Comment:
is there a reason to create a new tag spec? why not add `form` as an attribute to `input-common-attrs`?

https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L2667

This is probably why the test is failing. It's testing against the first input tagspec.

Reply:
Resolving as I believe there shouldn't be a new tagspec for this but included in input-common-attr attribute list.